pandas/tests/extension/test_categorical.py

```diff
@@ -207,6 +207,13 @@ def test_cast_nan_to_int(self, cls, values):
         with pytest.raises((ValueError, TypeError), match=msg):
             s.astype(int)
 
+    def test_cast_category_to_interval(self):
```

Comment:
Can you parametrize this over other dtypes that'd also hit the new code path (period, nullable integer, etc.)?